hedyorg/hedy
# [TESTING, CYPRESS] Tests Teacher page

**Description**

The first tests for the teacher page that test creating classes, duplicating classes and logging in as a teacher.

**Fixes _issue or discussion number_**

Fixes #3560 

**How to test**

Run 
`export ADMIN_USER=admin_user; python app.py` in your virtual environment. This ensures that there is an active admin_user and Hedy is running.

Then you can go to the /tests folder and run: 

`npx cypress run --spec cypress/e2e/teacher_page`

All specs should have passed!
